packages/shared/lib/core/router/subrouters/import-router.ts

```diff
@@ -69,7 +69,22 @@ export class ImportRouter extends Subrouter<ImportRoute> {
 
                 this.importFile = file
                 this.importFilePath = filePath
-                nextRoute = ImportRoute.BackupPassword
+
+                if (get(this.importType) === ImportType.SeedVault) {
```

Comment:
Line 64 changes the importType and we only want the migrationRequired error to be checked if the importType is stronghold, right? I've changed the condition to be broader
```suggestion
                if (get(this.importType) !== ImportType.Stronghold) {
```